app/validation.py

```diff
@@ -155,27 +155,21 @@ def get_validation_errors(validator_name, json_data,
 
 
 def min_price_less_than_max_price(error_map, json_data):
-
-    def return_min_and_max_price_keys(json_data):
-        if 'priceMin' in json_data:
-            return 'priceMin', 'priceMax'
-
-        specialist_price_keys = [k for k in json_data.keys() if k.endswith(('PriceMin', 'PriceMax'))]
-        if specialist_price_keys:
-            return \
-                next((key for key in specialist_price_keys if key.endswith('PriceMin')), None), \
-                next((key for key in specialist_price_keys if key.endswith('PriceMax')), None)
-
-        return None, None
-
-    min_price_key, max_price_key = return_min_and_max_price_keys(json_data)
-
-    if min_price_key:
-        if min_price_key in json_data and json_data.get(max_price_key):
-            if min_price_key not in error_map and max_price_key not in error_map:
-                if Decimal(json_data[min_price_key]) > Decimal(json_data[max_price_key]):
-                    return {max_price_key: 'max_less_than_min'}
-    return {}
+    price_errors = {}
+    for key in json_data.keys():
+        if key.lower().endswith('pricemin'):
+            prefix = key[:-8]
+            max_price_key = prefix + ('PriceMax' if prefix else 'priceMax')
+            if (
+                max_price_key in json_data and
+                json_data.get(max_price_key, None) and
```

Comment:
We don't need `max_price_key in json_data` check since we're doing a `.get` right after